AwesomeProject/screens/Menu/Menu.screen.js

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
+import {NavigationContainer} from '@react-navigation/native';
+import {Button, View, ScrollView} from 'react-native';
+import styles from '../Menu/Menu.styles';
+import AboutMe from '../AboutMe/AboutMe.screen';
+
+export default Screens = () => (
+  <NavigationContainer>
```

Comment:
navigation should be in nav.js file. This file should be inside nav folder on the same level with 'screens' folder.